app/src/org/commcare/logging/analytics/UpdateStats.java

```diff
@@ -62,6 +64,7 @@ public void resetStats(CommCareApp app) {
         startInstallTime = new Date().getTime();
         resourceInstallStats.clear();
         restartCount = 0;
+        resetCounter = 0;
```

Comment:
What's the logic behind splitting both of these out rather than just changing what hits restartCount?

Reply:
the restart counter gets used in determining the [retry back off time](https://github.com/dimagi/commcare-android/blob/commcare_2.48/app/src/org/commcare/engine/resource/AndroidResourceManager.java#L250), this code has been removed in `master` since the retry time is managed by the `WorkManager` and hence I will just maintain one counter when I port it in `master`